### PR TITLE
Adds flag to allow disabling the manifest processing

### PR DESCRIPTION
**Pull request by plaflamme**
Fixes https://github.com/thesamet/sbt-protoc/issues/249

### CODE REVIEW

File: src/main/scala/sbtprotoc/ProtocPlugin.scala [hunk 1]
```diff
@@ -304,6 +310,7 @@ object ProtocPlugin extends AutoPlugin {
       PB.targets := Nil,
       PB.protoSources := Nil,
       PB.protoSources += sourceDirectory.value / "protobuf",
+      PB.disableManifestProcessing := false,
```
Comment (bjaglin) [inline src/main/scala/sbtprotoc/ProtocPlugin.scala:313]: for other features, opt-out is achieved by overriding a setting set to true by default
https://github.com/thesamet/sbt-protoc/blob/99ce172205bc3479a412ca1d3afe497095a6f772/src/main/scala/sbtprotoc/ProtocPlugin.scala#L177-L179
what about:
```suggestion
      PB.manifestProcessing := true,
```

Comment (thesamet) [reply src/main/scala/sbtprotoc/ProtocPlugin.scala:313]: +1 - for consistency with other options.

Comment (plaflamme) [reply src/main/scala/sbtprotoc/ProtocPlugin.scala:313]: Yes, I think this is a better approach. I was following the suggestion @thesamet made [here](https://github.com/thesamet/sbt-protoc/issues/249#issuecomment-857204380). Which one do you guys prefer?

Comment (plaflamme) [reply src/main/scala/sbtprotoc/ProtocPlugin.scala]: Ok, I've added https://github.com/thesamet/sbt-protoc/pull/250/commits/ace440953891b81de9bed2a0a280092c22a98274 to address this

**Pull request merged**